packet.go

```diff
@@ -74,7 +74,7 @@ type PacketAdaptationExtensionField struct {
 }
 
 // parsePacket parses a packet
-func parsePacket(i *astikit.BytesIterator) (p *Packet, err error) {
+func parsePacket(i *astikit.BytesIterator, flt PacketFilter) (p *Packet, err error) {
```

Comment:
Could your rename the variable from `flt` to `s` ?

Reply:
Done